query-engine/connectors/mongodb-query-connector/Cargo.toml

```diff
@@ -21,8 +21,9 @@ tracing = "0.1"
 tracing-futures = "0.2"
 uuid.workspace = true
 indexmap = "1.7"
-query-engine-metrics = {path = "../../metrics"}
+query-engine-metrics = { path = "../../metrics" }
 cuid = { git = "https://github.com/prisma/cuid-rust", branch = "wasm32-support" }
+derive_more = "0.99.17"
```

Comment:
I'm a bit meh on the idea of bringing a new crate for the single purpose of avoiding a couple of `Self::new` methods. I see it has a bunch of other macros that could be useful. Let's try it.

Reply:
I agree. We have used this crate a lot for other purposes in my previous project, so I also hope that it will come more useful than just for this.